docs/api-reference.md

```diff
@@ -447,4 +447,27 @@ A vertically-centered Quote layout for if you want to present a quote and attrib
 | `quote`               | `string | ReactNode`                        | ✅        | `To be, or not to be`  |
 | `attribution`         | `string | ReactNode`                        | ✅        | `William Shakespeare`  |
 | `quoteProps`          | [Text Props](#typography-tags)  | ❌        | { fontSize: "100px" }  |
-| `attributionProps`    | [Text Props](#typography-tags)  | ❌        | { fontSize: "48px" }   |
\ No newline at end of file
+| `attributionProps`    | [Text Props](#typography-tags)  | ❌        | { fontSize: "48px" }   |
+
+### `SlideLayout.SingleCodeLayout`
+
+A layout with a single code pane and an optional title for if you want one code block per slide.
+
+| Props              | Type                                | Required | Example                         |
+|--------------------|-------------------------------------|----------|---------------------------------|
+| `...slideProps`    | [Slide Props](#slide)               | ❌        |                                 |
+| `title`            | `string`                            | ❌        | `Show me the code!`             |
+| `titleProps`       | [Heading Props](#typography-tags)   | ❌        | `{ color: 'red' }`              |
+| `children`         | `string`                            | ✅        | `const Component = (props: componentProps): JSX.Element = {...}` |
+| `language`         | `boolean`                           | ✅        | `false`                          |
+
+### `SlideLayout.MultiCodeLayout`
+
+A layout with multiple code panes and optional descriptions, and an optional title for if you want more than one code block per slide or code with description text.
+
+| Props              | Type                                | Required | Example                         |
+|--------------------|-------------------------------------|----------|---------------------------------|
+| `...slideProps`    | [Slide Props](#slide)               | ❌        |                                 |
+| `title`            | `string`                            | ❌        | `Show me the code!`             |
+| `titleProps`       | [Heading Props](#typography-tags)   | ❌        | `{ color: 'red' }`              |
+| `codePaneProps`    | `PropTypes.arrayOf(PropTypes.object([Code Pane props](#code-pane)))` | ✅        | `[{ code: 'console.log("hello world!")', language: 'jsx', description: 'Say hello', props: {...} }, {...}]` |
```

Comment:
I'm unsure if the current Type column for `codePaneProps` is the best way to document the type for this prop.